lib/cable_ready/broadcaster.rb

```diff
@@ -5,12 +5,122 @@ module Broadcaster
     include Identifiable
     extend ::ActiveSupport::Concern
 
+    included do |base|
```

Comment:
This thought occurred to me, too.

`CableReady::ModelBroadcaster` ?

Open to suggestions from the crowd.

Reply:
Wouldn't be opposed to `Broadcastable`. Also, it could live in `app/models/concerns` for consistency with other railties (such as Turbo)

Reply:
`Broadcastable` sounds good to me, with the possible exception that it's so close to `Broadcaster` that it might cause confusion. What about `CableReady::BroadcastFrom` or `CableReady::ModelStreamer` or something along those lines?

I do think that splitting these up is the right decision, even if it means some folks will include both concerns in their `ApplicationRecord`... I think that's fine, even if it's not as sexy.

Should we move the existing `broadcaster.rb` into `app/models/concerns`, too?

Reply:
Would go with `BroadcastFrom` then.

I was asking about putting it into `app/models/concerns` maybe because of autoloading... IIUC it would decouple it from the main `require` if we specify it to be `autoload`ed in the railtie definition.

The point being that it's optional functionality that doesn't need to go into memory by default

Reply:
That all makes good sense to me! I get it, now.

I'll see what I can do! ;)

Reply:
Newcomer to CableReady, so I don't know what `Broadcaster` currently does, but I like `Broadcastable` for the concern of the functionality in this PR, especially since there is another invocation required to actually _enable_ broadcasting (e.g. `enable_broadcasts`). No strong feelings on the naming, though... just answering the call for feedback. :-)

Reply:
Oh, I'm glad you asked! `CableReady::Broadcaster` is the primary way that you put CR to work in your application. You include this anywhere in your app, and you're ready to party: https://github.com/stimulusreflex/cable_ready/blob/master/lib/cable_ready/broadcaster.rb

Looking at it in the context of our work together not being present, you can see that it's short, simple and direct.

When we look at https://github.com/stimulusreflex/cable_ready/blob/a42981d28b9724c687dc31aeecbac7dfc2b6ddf1/lib/cable_ready/broadcaster.rb it just blows up the original intent of the module. Even though I take pains to only include the broadcasts stuff in ActiveRecord classes, there's still a ton of stuff that gets brought along for the ride and Julian's right to worry about the footprint.

Moving it to a concern that can be autoloaded on demand is something a good citizen would do.

Upon consideration, I'm okay with `Broadcastable`. @hopsoft will still have to have final say.